extensions/csv/src/main/java/io/deephaven/csv/CsvTools.java

```diff
@@ -219,18 +220,12 @@ public static Table readCsv(String path, CsvSpecs specs) throws CsvReaderExcepti
     @ScriptApi
     public static Table readCsv(InputStream stream, CsvSpecs specs) throws CsvReaderException {
         final CsvReader.Result result = CsvReader.read(specs, stream, makeMySinkFactory());
-        final String[] columnNames = result.columnNames();
-        final Sink<?>[] sinks = result.columns();
-        final Map<String, ColumnSource<?>> columns = new LinkedHashMap<>();
-        long maxSize = 0;
-        for (int ii = 0; ii < columnNames.length; ++ii) {
-            final String columnName = columnNames[ii];
-            final MySinkBase<?, ?> sink = (MySinkBase<?, ?>) sinks[ii];
-            maxSize = Math.max(maxSize, sink.resultSize());
```

Comment:
Is there any reason why tracking maxSize by each sink is different than result.numRows()?

Reply:
No, I was probably confused at the time of writing that code.